core/src/main/java/org/elasticsearch/action/admin/indices/rollover/RolloverResponse.java

```diff
@@ -39,25 +39,31 @@
     private static final String DRY_RUN = "dry_run";
     private static final String ROLLED_OVER = "rolled_over";
     private static final String CONDITIONS = "conditions";
+    private static final String CREATE_INDEX_ACKNOWLEDGED = "create_index_acknowledged";
```

Comment:
> it maybe then that with rollover the shards are acked but not the request. If we want to avoid that , I will only wait for shards active after the alias change is done.

That's kind of why its a bit nebulous.  The semantics of `acknowledged` on `ClusterStateUpdateResponse` should be the same, but the trick here is that we have two different cluster state updates - one for index creation and one for the alias.  We can move the shard waiting to after the alias swap, but then we also need to change the `MetaDataCreateIndexService` to make `onlyCreateIndex` public, and probably change the names.... `createIndex` only creates the index, `createIndexAndWait` includes the waiting... if they are to be public methods.. thoughts?